src/stan-users-guide/simple_linear_regression/regression_efficient.stan

```diff
@@ -0,0 +1,19 @@
+data {
+  int<lower=0> N;
```

Comment:
Space around all operators.  That's `=` and `*` in this example.

Reply:
I missed the *, but kept the parameter constraint because I thought it was standard.

The usage in the users-guide is inconsistent:

```
stan-users-guide$ grep '<lower=' *.Rmd | wc -l
     324
stan-users-guide$ grep '<lower =' *.Rmd | wc -l
      45

```
Spaces matches R guidelines. I'll change it as I go but I am concerned that Andrew won't like it.

Reply:
Yes, the user's guide inconsisent.  I think it should have spaces everywhere.  I think @andrewgelman is OK with that, but we can ask.  He usually sees these pings.